web/src/message_list_view.js

```diff
@@ -1807,7 +1807,7 @@ export class MessageListView {
         }
         this.sticky_recipient_message_id = message.id;
         const time = new Date(message.timestamp * 1000);
-        const rendered_date = timerender.render_date(time);
+        const rendered_date = util.the(timerender.render_date(time));
         dom_updates.html_updates.push({
             $element: $sticky_header.find(".recipient_row_date"),
             rendered_date,
```

Comment:
It seems to me like it might want the actual DOM element, and a much later redefinition of `rendered_date` might want the string, but doing  `.the` in `timerender.render_date` seems like the most rational thing to do with this detail.

Reply:
Moving the `.the` into `render_date` is a great idea, thanks!